servers/movie_writer/movie_writer_mjpeg.cpp

```diff
@@ -56,6 +56,17 @@ Error MovieWriterMJPEG::write_begin(const Size2i &p_movie_size, uint32_t p_fps,
 
 	base_path += ".avi";
 
+	if (GLOBAL_GET("editor/movie_writer/create_new_file_if_existing")) {
```

Comment:
This is exactly from the original [PR](https://github.com/godotengine/godot/pull/71420/files#diff-a3b0961b32d318c008e114afa1591c41fa4fb8a802053f8e3b6440893db28838R59)